cmd/clusterdeployment/list.go

```diff
@@ -14,6 +13,8 @@ import (
 	"github.com/openshift/osdctl/pkg/printer"
 )
 
+const hiveVersionMajorMinorPatchLabel string = "hive.openshift.io/version-major-minor-patch"
```

Comment:
This constant was the only reason we have hive as a dependency. I acknowledge that this does represent some risk as Hive could choose to change this tag and we would be none the wiser, but this command has rarely been used since the initial v3 --> v4 migration.